test/index.test.js

```diff
@@ -1324,10 +1317,94 @@ jobs:
 
             githubMock.repos.editHook.yieldsAsync(testError);
 
+            return scm.addWebhook(webhookConfig).then(assert.fail, (err) => {
+                assert.equal(err, testError);
+            });
+        });
+    });
+
+    describe('_getOpenedPrs', () => {
+        const scmUri = 'github.com:111:branchName';
+        const config = {
+            scmUri,
+            token: 'token'
+        };
+
+        beforeEach(() => {
+            githubMock.repos.getById.yieldsAsync(null, {
+                full_name: 'repoOwner/repoName'
+            });
+        });
+
+        it('returns a list of opened pull requests', () => {
+            githubMock.pullRequests.getAll.yieldsAsync(null, [
+                { number: 1 },
+                { number: 2 }
+            ]);
+
             /* eslint-disable no-underscore-dangle */
-            return scm._addWebhook(webhookConfig).then(assert.fail, (err) => {
+            return scm._getOpenedPrs(config).then((data) => {
             /* eslint-enable no-underscore-dangle */
-                assert.equal(err, testError);
+                assert.deepEqual(data, [
+                    {
+                        name: 'PR-1',
+                        ref: 'pull/1/merge'
+                    },
+                    {
+                        name: 'PR-2',
+                        ref: 'pull/2/merge'
+                    }
+                ]);
+
+                assert.calledWith(githubMock.repos.getById, { id: '111' });
+                assert.calledWith(githubMock.pullRequests.getAll, {
+                    owner: 'repoOwner',
+                    repo: 'repoName',
+                    state: 'open'
+                });
+            });
+        });
+
+        it('returns an empty list of pull requests', () => {
```

Comment:
Why we need this test here? Looks like it is just testing if `map` function works with empty array